RFCs/006-DynamicData-Attribute.md

```diff
@@ -0,0 +1,66 @@
+# RFC 006- DynamicData Attribute for Data Driven Tests
+
+## Summary
+This details the MSTest V2 framework attribute "DynamicData" for data driven tests where test data, declared as properties or methods, can be shared between more than one test cases.
```

Comment:
Should this be called `DynamicDataRow` to keep it conceptually aligned with `DataRow`?